src/sdk/cart/validate.ts

```diff
@@ -58,15 +58,42 @@ export const ValidateCartMutation = gql`
         name
       }
       gtin
+      additionalProperty {
+        propertyID
+        name
+        value
+        valueReference
+      }
     }
   }
 `
 
 export const isGift = (item: CartItem) => item.price === 0
 
+const serializeAdditionalProperty = (itemOffered: CartItem['itemOffered']) => {
```

Comment:
Can't we do something like:
```suggestion
const serializeAdditionalProperty = (itemOffered: CartItem['itemOffered']) => 
    itemOffered.additionalProperty?.map(({ propertyID }) => propertyID).join('-')
```